activejob/lib/active_job/railtie.rb

```diff
@@ -34,6 +34,12 @@ class Railtie < Rails::Railtie # :nodoc:
       ActiveSupport.on_load(:action_dispatch_integration_test) do
         include ActiveJob::TestHelper
       end
+
+      ActiveSupport.on_load(:active_record) do
+        active_record_options = app.config.active_record
+        active_record_options.destroy_association_later_job = ActiveRecord::DestroyAssociationLaterJob
+        active_record_options.destroy_later_job = ActiveRecord::DestroyLaterJob
```

Comment:
@rafaelfranca Is this the correct place and way to do this? Is there a good way to test this setting?

Reply:
Yes, that is the correct way, but you don't need to add the config to `app.config.active_record` anymore. You are already inside `ActiveRecord::Base` here. So this code can be:

```ruby
self.destroy_association_later_job = ActiveRecord::DestroyAssociationLaterJob
self.destroy_later_job = ActiveRecord::DestroyLaterJob
```

To test it you will need an integration test in the railties folder. Similar to those tests here. https://github.com/rails/rails/blob/master/railties/test/application/configuration_test.rb#L423-L432